lib/config/options/index.ts

```diff
@@ -522,8 +522,7 @@ const options: RenovateOptions[] = [
   },
   {
     name: 'dependencyDashboardOSVVulnerabilitySummary',
-    description:
-      'These labels will always be applied on the Dependency Dashboard issue, even when they have been removed manually.',
+    description: 'Set this option to display CVEs affecting dependencies.',
```

Comment:
```suggestion
    description: 'Control if the Dependency Dashboard issue shows CVEs.',
```

The description should explain _where_ the CVEs will be shown.

Reply:
IMO it's clear that the dependency dashboard is the location where it is displayed as the option is called `dependencyDashboardOSVVulnerabilitySummary`, but fine for me. 🤷

Reply:
I like to be clear, even if that means "pointing out the obvious". But yeah, the name is kinda self-explanatory. 😄